core/domain/caching_services.py

```diff
@@ -247,10 +258,10 @@ def delete_multi(namespace, sub_namespace, obj_ids):
     memcache_keys = [
         _get_memcache_key(namespace, sub_namespace, obj_id)
         for obj_id in obj_ids]
-    return memory_cache_services.delete_multi(memcache_keys) == len(obj_ids)
+    return bool(memory_cache_services.delete_multi(memcache_keys) == len(obj_ids)) # type: ignore[attr-defined]
```

Comment:
Why is the `bool` needed?

Reply:
before: it gives error of returning `Any` value form function that's why i added `bool` fuction and attribute not defined error.
but after we imported `memory_cache_services` from my_imports it fixed the problem